### DISCUSSION TITLE
IPv6 based UI for Rabbitmq version-13.12.12 is not opening

### DISCUSSION THREAD

**Discussion by pritam-d29** [rabbitmq/rabbitmq-server#12060] [Questions] [answered]
Hi Team,

Seek support in one of the issues, that I am facing with Rabbitmq version-13.12.12. 
The rabbitmq here I am using with my Kubernetes-1.28.8 in IPv6 env. 
I am deploying the rabbitmq pod with helm based cli.

Here post successful deployment as well the UI is not Opening for me on port 15672. 
All necessary changes have been made wrt the documentation meant for IPv6 but no luck.

Seek your help in resolving this as I am stuck in this issue for last couple of weeks. If anything needed then do let me know. 

Regards

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.12 is [out of commercial support](https://www.rabbitmq.com/release-information). 

There is a guide that explains [what the core team recommends for narrowing down connectivity issues](https://www.rabbitmq.com/docs/troubleshooting-networking).
